### ISSUE TITLE
Getting error "Must provide schema"

### ISSUE THREAD

**Issue by cramatt** [apollostack/subscriptions-transport-ws#58]

**Comment by helfer** [comment]
@cramatt unfortunately I can't help without a better error description. It could be that you're not providing a schema to your server's graphql-subscription code.

**Comment by cramatt** [comment]
Thanks @helfer. Something odd happened to this issue because I recall writing a much more verbose issue with an example. And then I recall closing the issue because I figured out the problem. Maybe my memory is not working as expected. 

Anyhow, I traced this to initializing the WebSocketManger with options.Schema instead of options.schema. The uppercase was not being recognized hence the error. 

Once I changed my options it worked as expected. 

I had mentioned before it might be worth adding a debug message to https://github.com/apollostack/graphql-subscriptions but thats an issue I would consider opening there.
